jaxrs/src/main/java/se/fortnox/reactivewizard/jaxrs/JaxRsRequest.java

```diff
@@ -108,8 +108,14 @@ public String getQueryParam(String key) {
     public String getQueryParam(String key, String defaultValue) {
         if (queryParameters == null) {
             QueryStringDecoder queryStringDecoder = new QueryStringDecoder(req.uri());
-            queryParameters = queryStringDecoder.parameters();
+            try {
+                queryParameters = queryStringDecoder.parameters();
+            } catch (IllegalArgumentException e) {
+                LOG.error(e.getMessage());
```

Comment:
We should have a log message that briefly explains what happened and then include the exception as the last param in LOG.error. Thus we will get the message and stacktrace.